utils/runtime.go

```diff
@@ -0,0 +1,61 @@
+package utils
+
+import (
+	"context"
+	sharedpb "github.com/tron-us/go-btfs-common/protos/shared"
+	"github.com/tron-us/go-common/constant"
+	"github.com/tron-us/go-common/db"
+	"github.com/tron-us/go-common/db/postgres"
+	"github.com/tron-us/go-common/db/redis"
+	"github.com/tron-us/go-common/log"
+
+	"go.uber.org/zap"
+)
+
+func CheckRuntime(ctx context.Context, runtime *sharedpb.SignedRuntimeInfoRequest, connection db.ConnectionUrls) (*sharedpb.RuntimeInfoReport, error) {
```

Comment:
The function is hard coupled, and use an assumption that we will only have one of PostGre and Redis, if we have used both, the function cannot support. Suggest refer Facade pattern for the runtime response.

Reply:
N/A

Reply:
n/a